lib: agps: remove deprecated library (#7341)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/nrf_cloud_pgps.rst

* :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_FRAGMENT_SIZE`
* :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT`

Configure the following option if you need your application to use A-GPS as well, for coarse time and position data and to get the fastest TTFF:

* :kconfig:option:`CONFIG_NRF_CLOUD_AGPS`

> If A-GPS is not desired (due to data costs, low power requirements, or expected frequent loss of cloud connectivity), the option above must be disabled.

Reviewer: ```suggestion
If A-GPS is not desired (due to data costs, low power requirements, or expected frequent loss of cloud connectivity), you must disable the option.
```

Author: Updated.
